deis-cluster-coreos/azuredeploy.json

```diff
@@ -359,8 +366,19 @@
             "publisher": "[variables('imagePublisher')]",
             "offer": "[variables('imageOffer')]",
             "sku": "[variables('imageSKU')]",
-            "version": "latest"
+            "version": "647.2.0"
```

Comment:
This version might get removed from the image repository in the future at which point the template will break. Is there a reason for using a specific version rather than latest?